CHANGELOG.md

```
@@ -47,8 +52,8 @@ All notable changes to this project are documented here. The format follows
   with `.optional()` round-tripping.
 - `checkModels` and the `openrouter-zdr check-model` CLI.
 
-[Unreleased]: https://github.com/FlorianRiquelme/openrouter-zdr/compare/v0.1.3...HEAD
+[Unreleased]: https://github.com/FlorianRiquelme/openrouter-zdr/compare/v0.1.4...HEAD
```

markdownlint-cli2 (0.23.2)

[warning] Link and image reference definitions should be needed
Unused link or image reference definition: "unreleased"

(MD053, link-image-reference-definitions)